lib/gringotts/gateways/global_collect.ex

```diff
@@ -433,7 +433,7 @@ defmodule Gringotts.Gateways.GlobalCollect do
     datetime = Timex.now |> Timex.local
     strftime_str = Timex.format!(datetime, "%a, %d %b %Y %H:%M:%S ", :strftime)
     time_zone = Timex.timezone(:local, datetime)
-    time = strftime_str <>"#{time_zone.abbreviation}"
+    time = strftime_str <> "#{time_zone.abbreviation}"
```

Comment:
`time` is unused and `strftime_str <> "#{time_zone.abbreviation}"` need not be bound to it.